spec/lib/rails_ext/action_dispatch/debug_exceptions_spec.rb

```diff
@@ -2,6 +2,24 @@
 require "rails"
 require "govuk_app_config/rails_ext/action_dispatch/debug_exceptions"
 
+class NoMethodTestClass; end
+
+class WrongParametersTestClass
+private
+
+  def log_error(_different, _parameters); end
+end
+
+class RightParametersTestClass
+private
+
+  def log_error(request, wrapper); end
+end
+
+class FakeDebugExceptions
+  def log_error(request, wrapper); end
+end
```

Comment:
I think these classes can be defined by lets too

Reply:
Good point, I've fixed that in e8a04dd6be5854056dfb7bf4a3cb259fdeb8142a.